### DISCUSSION TITLE
dynamic column placeholder

### DISCUSSION THREAD

**Discussion by teoconnect** [Budibase/budibase#9452] [Help]
Hello,
i need community help...
![2023-01-26-1417](https://user-images.githubusercontent.com/48184577/214884864-2242852e-8786-43a0-a7e8-1f89965510d2.png)
as you can see i have multiple db column named by generic number, i need to combine the first part of the repeater with a dynamic number provided by other value present in the app (like it was a join)
is possible?

![image](https://user-images.githubusercontent.com/48184577/214885544-15f56cd5-9dc8-4d78-ac87-7a6c0706148c.png)

thanks a lot

**Comment by melohagan** [comment]
Hey @teoconnect 

This looks a bit mad. I would probably use a custom SQL query, and transform the data first to make it more suitable for use in your app.

 - https://docs.budibase.com/docs/custom-queries
 - https://docs.budibase.com/docs/transformers
